### ISSUE TITLE
Update to 5.0.1: Table 'point_groups' already exists

### ISSUE THREAD

**Issue by Tie-fighter** [mautic/mautic#13192]
### Mautic Version

5.0.x series

### PHP version

8.0.30

### What browsers are you seeing the problem on?

Not relevant

### What happened?

I upgraded my mautic instance via console.
Running console mautic:update:apply went fine, mautic:update:apply --finish threw an error.

Seems like the database migration cannot be applied because the table already exists.

### How can we reproduce this issue?

Step 1: Install 5.0.x
Step 2: Update to 5.0.1 via console

### Relevant log output

```shell
[2024-01-10 13:11:11] mautic.ERROR: Doctrine\DBAL\Exception\TableExistsException: An exception occurred while executing a query: SQLSTATE[42S01]: Base table or view already exists: 1050 Table 'point_groups' already exists (uncaught exception) at /var/www/mautic/vendor/doctrine/dbal/src/Driver/API/MySQL/ExceptionConverter.php line 45 while running console command `doctrine:migrations:migrate` [stack trace] #0 /var/www/mautic/vendor/doctrine/dbal/src/Connection.php(1935): Doctrine\DBAL\Driver\API\MySQL\ExceptionConverter->convert() #1 /var/www/mautic/vendor/doctrine/dbal/src/Connection.php(1878): Doctrine\DBAL\Connection->handleDriverException() #2 /var/www/mautic/vendor/doctrine/dbal/src/Connection.php(1103): Doctrine\DBAL\Connection->convertExceptionDuringQuery() #3 /var/www/mautic/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(298): Doctrine\DBAL\Connection->executeQuery() #4 /var/www/mautic/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(173): Doctrine\Migrations\Version\DbalExecutor->executeResult() #5 /var/www/mautic/vendor/doctrine/migrations/lib/Doctrine/Migrations/Version/DbalExecutor.php(96): Doctrine\Migrations\Version\DbalExecutor->executeMigration() #6 /var/www/mautic/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(107): Doctrine\Migrations\Version\DbalExecutor->execute() #7 /var/www/mautic/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(71): Doctrine\Migrations\DbalMigrator->executePlan() #8 /var/www/mautic/vendor/doctrine/migrations/lib/Doctrine/Migrations/DbalMigrator.php(157): Doctrine\Migrations\DbalMigrator->executeMigrations() #9 /var/www/mautic/vendor/doctrine/migrations/lib/Doctrine/Migrations/Tools/Console/Command/MigrateCommand.php(211): Doctrine\Migrations\DbalMigrator->migrate() #10 /var/www/mautic/vendor/symfony/console/Command/Command.php(298): Doctrine\Migrations\Tools\Console\Command\MigrateCommand->execute() #11 /var/www/mautic/vendor/symfony/console/Application.php(1058): Symfony\Component\Console\Command\Command->run() #12 /var/www/mautic/vendor/symfony/framework-bundle/Console/Application.php(96): Symfony\Component\Console\Application->doRunCommand() #13 /var/www/mautic/vendor/symfony/console/Application.php(301): Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() #14 /var/www/mautic/vendor/symfony/framework-bundle/Console/Application.php(82): Symfony\Component\Console\Application->doRun() #15 /var/www/mautic/vendor/symfony/console/Application.php(171): Symfony\Bundle\FrameworkBundle\Console\Application->doRun() #16 /var/www/mautic/app/bundles/CoreBundle/Update/Step/UpdateSchemaStep.php(51): Symfony\Component\Console\Application->run() #17 /var/www/mautic/app/bundles/CoreBundle/Command/ApplyUpdatesCommand.php(137): Mautic\CoreBundle\Update\Step\UpdateSchemaStep->execute() #18 /var/www/mautic/app/bundles/CoreBundle/Command/ApplyUpdatesCommand.php(99): Mautic\CoreBundle\Command\ApplyUpdatesCommand->finishUpgrade() #19 /var/www/mautic/vendor/symfony/console/Command/Command.php(298): Mautic\CoreBundle\Command\ApplyUpdatesCommand->execute() #20 /var/www/mautic/vendor/symfony/console/Application.php(1058): Symfony\Component\Console\Command\Command->run() #21 /var/www/mautic/vendor/symfony/framework-bundle/Console/Application.php(96): Symfony\Component\Console\Application->doRunCommand() #22 /var/www/mautic/vendor/symfony/console/Application.php(301): Symfony\Bundle\FrameworkBundle\Console\Application->doRunCommand() #23 /var/www/mautic/vendor/symfony/framework-bundle/Console/Application.php(82): Symfony\Component\Console\Application->doRun() #24 /var/www/mautic/vendor/symfony/console/Application.php(171): Symfony\Bundle\FrameworkBundle\Console\Application->doRun() #25 /var/www/mautic/bin/console(16): Symfony\Component\Console\Application->run() #26 {main} [] {"hostname":"[redacted]","pid":59864}
[2024-01-10 13:11:11] mautic.WARNING: Command `doctrine:migrations:migrate` exited with status code 255 [] {"hostname":"[redacted]","pid":59864}
[2024-01-10 13:11:11] mautic.WARNING: Command `mautic:update:apply` exited with status code 1 [] {"hostname":"[redacted]","pid":59864}
```

### Code of Conduct

- [X] I confirm that I have read and agree to follow this project's Code of Conduct

<br /><hr>
Care about this issue? Want to get it resolved sooner? If you are a <a href='https://www.mautic.org/become-a-member-of-mautic'>member of Mautic</a>, you can add some funds to the <a href='https://opencollective.com/mautic/projects/bounties'>Bounties Project</a> so that the person who completes this task can claim those funds once it is merged by a member of the core team! Read the docs <a href='https://contribute.mautic.org/product-team/mautic-bounty-programme'>here.</a>

**Comment by code5rick** [comment]
That issue happened with us too. We had to delete the database.

**Comment by Tie-fighter** [comment]
> That issue happened with us too. We had to delete the database.

I would like to avoid that.
Does dropping the table or exporting, migrating, importing help?

PS: Sounds to me like there is just an if exists missing.

**Comment by code5rick** [comment]
It was a brand new installation, and when we migrated from 5.0 to 5.0.1, that issue occurred.

**Comment by Tie-fighter** [comment]
The failing migration is "Version20230621074925"

**Comment by escopecz** [comment]
This is a duplicate of https://github.com/mautic/mautic/issues/12993. Let's discuss it there.

**Comment by Tie-fighter** [comment]
Did an doctrine:migration:execute --down "Mautic\Migrations\Version20230621074925" followed by --up and that seemss to have fixed it..
